features/kafka/producer/src/main/proto/opennms-kafka-producer.proto

```diff
@@ -147,3 +147,42 @@ message Node {
   repeated SnmpInterface snmp_interface = 12;
   HwEntity hw_inventory = 13;
 }
+
+message TopologyRef {
+  string id = 1;
+  string protocol = 2;
+}
+
+message TopologyVertexNode {
+  NodeCriteria node_criteria = 1;
+}
+
+message TopologyVertexSegment {
+}
+
+message TopologyPort {
+  string vertex_id = 1;
+  uint64 ifindex = 2; 
```

Comment:
The `ifIndex` and `ifName` fields are already called `if_index` and `if_name ` in some of the other existing messages - can use this same name format?